app/main.py

```diff
@@ -2,5 +2,15 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    results = {}
+
+    def wrap(*args) -> int:
+        if args in results:
+            print("Getting from cache")
+            return results[args]
```

Comment:
- Good to add an annotation from `functools `module
- Also, it should work with `**kwargs`. Create a unique key using args and kwargs

Reply:
I hope I understood comments
Added fixes